### PR TITLE
RLPNC-7496: Support specifying parameters + universes in Java Bindings

### PR DESCRIPTION
**Pull request by mrwrench24**

### CODE REVIEW

**Review by meganlevin — APPROVED** [review]
lgtm! just resolve those merge conflicts

**Comment by iredpath** [comment]
When this is done on our end, please tag Seth and Adam as reviewers
Sonarqube shows 0% coverage on the new lines, but there are new test cases with params/universe specified - hopefully we can just ignore this

File: model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityProperties.java [hunk 1]
```diff
@@ -25,8 +28,11 @@
 @Jacksonized
 @Builder
 @Value
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class RecordSimilarityProperties {
     @Builder.Default
     @NotNull @Valid Double threshold = 0.0;
-    @NotNull @Valid Boolean includeExplainInfo;
+    @NotNull @Valid boolean includeExplainInfo;
```
Comment (asoosprecognox) [inline model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityProperties.java]: This was just changed in https://github.com/rosette-api/java/pull/230/files#diff-18fd20221ce02b841d506fb46553722a66e9ee47fc7ecfe89b598deeb1bd70aa to Boolean. Is it intentionally changed back to boolean or was it the result of the merge?

**Review by asoosprecognox — COMMENTED** [review]
When it comes to the coverage numbers I think its fine. 
@seth-mg  @Katsuya-Tomioka It seems like lombok and Jackson needs some extra configuration to register the coverage correctly. (https://stackoverflow.com/questions/74592390/how-to-cover-100-sonarqube-code-coverage-on-java-bean-class-with-getter-build and https://stackoverflow.com/questions/44584487/improve-lombok-data-code-coverage). Should this be added?

File: model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityProperties.java (hunk 1 above)
Comment (iredpath) [reply model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityProperties.java]: Thanks for that catch Adam, we want this to stay a `Boolean` for other work and it must've changed back as part of the merge.  @mrwrench24 please set it back to `Boolean`

Comment (mrwrench24) [reply model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityProperties.java]: Just fixed, thanks for pointing this out, my bad

**Comment by iredpath** [comment]
@asoosprecognox @seth-mg @Katsuya-Tomioka is anything else needed from us to get this merged in?

**Comment by asoosprecognox** [comment]
No, lets merge

**Pull request merged**